test/plugins/features/movies/controller.test.js

```diff
@@ -90,5 +90,63 @@ describe('movie controller', () => {
 
   });
 
+  describe('add', () => {
+
+    const movie_list = [
```

Comment:
If this is the same for each test in this file, it can be moved up to higher level and not repeated. Same with the beforeEach block